packages/astro/src/assets/fonts/utils.ts

```diff
@@ -316,3 +302,8 @@ export function familiesToUnifontProviders({
 
 	return { families, providers };
 }
+
+/** Checks if the name starts with --, doesn't include a space nor a colon */
+export function isValidCssVariableName(name: string) {
+	return name.startsWith('--') && !name.includes(' ') && !name.includes(':');
```

Comment:
CSS custom properties [names](https://developer.mozilla.org/en-US/docs/Web/CSS/var#custom-property-name) are defined with an [identifier](https://developer.mozilla.org/en-US/docs/Web/CSS/ident) that includes a a few more restrictions if we want to be more thorough in the validation?

Reply:
Ah thanks I couldn't find the docs for those

Reply:
Do you know if there are libraries out there for this? We use `cssesc` for VT but AFAIK it's for creating identifiers only

Reply:
Dedicated to validating CSS ident? Not something I personally know of or had the need before.

Reply:
I guess it’s probably OK to be a bit loose with the validation and document as `should be a valid [ident](link to MDN)` rather than trying to recreate full CSS spec compatibility here.